bowtie.wdl

```diff
@@ -0,0 +1,43 @@
+task Bowtie_se {
+    File reference
+    File read1
+    String sam
+    String? precommand
+    Int? threads
```

Comment:
You added threads here, but did not include a runtime section that uses the threads.
```WDL
runtime {
  cpu: ${default=1 threads}
}
```
You can also set the default here
```WDL
  Int? threads = 1
```
Which looks cleaner. But I have had issues with overriding this default from the config file. I should file a bug for that.